src/editor/src/UI.cpp

```diff
@@ -26,7 +29,8 @@ void Hush::UI::InitializePanels()
     S_ACTIVE_PANELS.push_back(CreatePanel<ScenePanel>());
     S_ACTIVE_PANELS.push_back(CreatePanel<HierarchyPanel>());
     S_ACTIVE_PANELS.push_back(CreatePanel<ContentPanel>());
-    S_ACTIVE_PANELS.push_back(CreatePanel<DebugUI>());
+	S_ACTIVE_PANELS.push_back(CreatePanel<DebugUI>());
```

Comment:
Continuing the previous comment, something like :
`this->mActivePanels.push_back(...);`

Also, since CreatePanel just does a make_unique, do we need a custom function for that?

Reply:
The idea would've been to not be dependent on an instance of this class and just use it as static functions, but we can take the route of making it an instance as a singleton

Reply:
As for the CreatePanel method, it infers some of the type information, just convenience